docker/Dockerfile

```diff
@@ -1,23 +1,29 @@
 ARG pg_version=12.2
 
-FROM postgres:$pg_version as is_jsonb_valid_build
+FROM postgres:$pg_version as extension_builder
 
-RUN apt-get update -y
-RUN apt-get install -y build-essential curl
+RUN apt-get update -y --fix-missing
```

Comment:
I don't think the `-y` is needed. It never had apt prompt me during the a update. Also not sure if `--fix-missing` does anything with update because it is not downloading any packages, see man page excerpt below.
```
-m, --ignore-missing, --fix-missing
    Ignore missing packages. If packages cannot be retrieved or fail the integrity check after retrieval (corrupted package files), hold back those packages and handle the result. Use of this option together with -f may produce an error in some situations. If a package is selected for installation (particularly if it is mentioned on the command-line) and it could not be downloaded then it will be silently held back. 
```

Reply:
fix-missing was required otherwise i got an execption because of missing packages
-y i guess is needed if there are new default packages

Reply:
I checked out your branch and removed `-y` and `--fix-missing` and was still able to build the container. `update` should not install any packages, it's only job is to update the index of available packages. `upgrade` is updating the packages itself, maybe that is what you wanted to do?
```
update
    Used to re-synchronize the package index files from their sources. The indexes of available packages are fetched from the location(s) specified in /etc/apt/sources.list(5). An update should always be performed before an upgrade or dist-upgrade. 
upgrade
    Used to install the newest versions of all packages currently installed on the system from the sources enumerated in /etc/apt/sources.list(5). Packages currently installed with new versions available are retrieved and upgraded; under no circumstances are currently installed packages removed, nor are packages that are not already installed retrieved and installed. New versions of currently installed packages that cannot be upgraded without changing the install status of another package will be left at their current version. An update must be performed first so that apt-get knows that new versions of packages are available. 
```

Reply:
no wanted to update, however it seems that has something to do with the previous cached layer, hm

Reply:
removed the options, now it works without